wgpu-core/src/lib.rs

```diff
@@ -100,7 +100,7 @@ impl RefCount {
 
 impl Clone for RefCount {
     fn clone(&self) -> Self {
-        let old_size = unsafe { self.0.as_ref() }.fetch_add(1, Ordering::Relaxed);
+        let old_size = unsafe { self.0.as_ref() }.fetch_add(1, Ordering::Release);
```

Comment:
read-modify-writes can't be only release, they either have to be relaxed or acq-rel

Reply:
Actually, [they can](https://doc.rust-lang.org/stable/std/sync/atomic/struct.AtomicI8.html#method.fetch_add):
> Note that using Acquire makes the store part of this operation Relaxed, and using Release makes the load part Relaxed.

Reply:
Wow, interesting. I wonder if this is a rust/c++ difference, because MSVC yelled at me for an "invalid memory ordering" when I did it in c++. Could just be a problem with MSVC's already questionable memory model.

Reply:
I bet we'll come back to this in #150 :)